pkg/operator/k8sutil/pod.go

```diff
@@ -119,6 +120,41 @@ func GetSpecContainerImage(spec v1.PodSpec, name string, initContainer bool) (st
 	return image.Image, nil
 }
 
+// Replaces the pod default toleration of 300s used when the node controller
+// detect a not ready node (node.kubernetes.io/unreachable)
+func AddUnreachableNodeToleration(podSpec *v1.PodSpec) {
+	// The amount of time for this pod toleration can be modified by users
+	// changing the value of <ROOK_UNREACHABLE_NODE_TOLERATION_SECONDS> Rook operator
+	// variable.
+	// Node controller will wait 40 seconds by default before mark a node as
+	// unreachable. After 40s + ROOK_UNREACHABLE_NODE_TOLERATION_SECONDS the pod
+	// will be scheduled in other node
+	// Only one <toleration> to <unreachable> nodes can be added
+	var tolerationSeconds int64 = 5
+	urTolerationDuration := os.Getenv("ROOK_UNREACHABLE_NODE_TOLERATION_SECONDS")
```

Comment:
nit: you can rename the var since it's seconds now instead of a general duration

Reply:
Changed.